econet_heatpump_water_heater.yaml

```diff
@@ -30,8 +37,7 @@ logger:
   tx_buffer_size: 2000
 
 external_components:
-  # - source: github://Daniel-dev22/esphome-econet
```

Comment:
Uncomment this and delete your new line

Reply:
Not sure I understand - do you just want to keep the local reference or you want something else?

Reply:
I'm saying the new line you added matches this commented out line. No reason to have both.